lib/config.ts

```diff
@@ -33,6 +33,14 @@ class Config {
 
   readonly enableErrLogging: boolean;
 
+  readonly adminAuthFile = 'playwright/.auth/admin.json';
+  readonly user1AuthFile = 'playwright/.auth/user1.json';
```

Comment:
Are those users with some specific roles ? maybe better naming for it if it is specific.

Reply:
AFAIR admin is admin, user_1 and user_2 are regular users, stratosphere users are specific users for billing tests. ANy suggestions with naming?